test2code/src/test/kotlin/com/epam/drill/test2code/coverage/BufferTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.epam.drill.test2code.coverage
+
+import com.epam.drill.common.agent.*
+import com.epam.drill.jacoco.*
+import com.nhaarman.mockitokotlin2.*
+import kotlinx.coroutines.*
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Assertions.*
+import java.util.*
+import java.util.concurrent.atomic.*
+import kotlin.random.Random
+
+class BufferTest {
```

Comment:
Current implementation is testing `CoverageSender` + `InMemoryBuffer` + `CoverageTransport`, not just the buffer itself. Suggest renaming to `CoverageRetentionTest`.

Reply:
done.